### ISSUE TITLE
Property of type List<String>

### ISSUE THREAD

**Issue by apuder** [satyan/sugar#535]
I created a SugarRecord with a property of List<String>. I can save it, but then when I try to find that SugarRecord, that property is null. I'm also not quite sure how the List<String> is stored in the sqlite DB. The column seems to be a string surrounded by '[' and ']' and each element is separated by a comma. That is not a legal sqlite data type, correct? (.schema in the command line tool sqlite3 does not show a type for that column).

Does Sugar support List<String>?

**Comment by jonisaa** [comment]
Show me the code of your class. The List<String> it's a relationship? 

I mean the code should work. Let me generate a sample in order to reproduce your error.

**Comment by JonatanSalas** [comment]
@apuder do you solve this?

**Comment by apuder** [comment]
My apologies. I have too many GH notifications. I was evaluating different DBs back in February and also tried Sugar. Let me see if I can still find my test code. But I never managed to solve the issue. I appreciate you following up. I'll add a comment to this issue if I can find my test code.

**Comment by JonatanSalas** [comment]
Okey, if you find it. Put your repository url so I can clone and check it. :)

**Comment by apuder** [comment]
I found my old test program. Just pushed it to GH:
git@github.com:apuder/SugarTest.git
In that version I tried to circumvent the List<String> issue. What I wanted to do is to add a list of categories to OrderedBook. Initially I tried ArrayList<String> as the type. That didn't work so I tried to introduce a wrapper class Category that just has one string-typed property. Something else wasn't working neither after which I stopped trying.
